src/app/page.tsx

```diff
@@ -1,9 +1,32 @@
-import ItemsList from './components/ItemsList'
+import Newsletter from 'components/newsletter'
+import ListItem from 'components/blogList'
```

Comment:
adjust this as it is confusing to import a component called "blogList" and use it as "ListItem"
It's very generic